usefulUtilities/sitV2/appResources/appData/sitServerV2.js

```diff
@@ -157,13 +156,44 @@
     }
 
     // #endregion CAN SIT ZONE
+
+    /* Check children of the sit cube to make sure there is only one zone. If more are found, delete them */
+    function checkForExtraZones() {
+        var canSitZones = [];
+        Entities.getChildrenIDs(_this.entityID).forEach(function(childOfSitCube) {
+            var name = Entities.getEntityProperties(childOfSitCube, 'name').name;
+            if (name && name.indexOf("canSitZone") > -1) {
+                canSitZones.push(childOfSitCube);
+            }
+        });
+        if (canSitZones.length > 1) {
+            canSitZones.pop();
+            canSitZones.forEach(function(canSitZone) {
+                Entities.deleteEntity(canSitZone);
+            });
+        }
+    }
+
+    /* Set a random timeout with exponential growth to check for extra zones */
+    var ONE_SECOND_MS = 1000;
+    var ONE_TENTH_SECOND_MS = 100;
+    var EXPONENT = 2;
```

Comment:
this isn't an exponent in this implementation, it's more like a multiplier. please change the name of this variable to something like `MULTIPLIER`.